### PR TITLE
Feature/#87 general help page

### PR DESCRIPTION
**Pull request by Princesso**
Adds a general getting started help page to MVP static pages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "whatwg-fetch": "2.0.3"
   },
   "scripts": {
-    "start": "NODE_PATH=src && node scripts/start.js",
+    "start": "NODE_PATH=src && node scripts/build.js",
```
Comment (innocentamadi) [inline package.json]: What? Why are we running the build script instead of the start script here?? Why this change?

Comment (Princesso) [reply package.json]: I didn't notice. Thanks for pointing that out

File: src/js/components/help/components/pages/backstage/scenes/event-guests/EventGuests.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import styled from 'styled-components'
+import Sidebar from '../../../../help-topics'
+import Topic from '../../../../topic'
+import Desrciption from '../../../../help-description'
+import Description from '../../../../help-description';
+import {Link} from 'react-router-dom'
+
+const StyledEventGuests=styled.div`
```
Comment (innocentamadi) [inline src/js/components/help/components/pages/backstage/scenes/event-guests/EventGuests.js]: There should be a space between the `=` and the words.
`StyledEventGuests = styled.div`

File: src/js/components/help/components/pages/backstage/scenes/event-guests/EventGuests.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+height: 500px;
+}
+
+.help-main-content{
+  font-size: 15px;
+  margin-left:80px;
+  max-width: 650px;
+}
+`
+const EventGuests = ()=>{
```
Comment (innocentamadi) [inline src/js/components/help/components/pages/backstage/scenes/event-guests/EventGuests.js]: Space needed before and after the `=>`

File: src/js/components/help/components/pages/backstage/scenes/event-guests/EventGuests.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            <br/><br/> Right after using the backstage button, you can find Guest related information on 
+            the guest section. <br/><br/> 
+            If your guests have indicated interest in the event through eventplog, you can find your guest
+            information on the all guest section of guests. You can directly check them in from here.
+            <br/><br/> 
+            You can also import your guest CSV if you have collected guest information on a different URL.
+            When you import the list, the guest information moves to the all guests section where you can
+            check them in.
+            <br/><br/> 
+            If there are new guest that have arrived at the event venue but did not register, you can register and
```
Comment (innocentamadi) [inline src/js/components/help/components/pages/backstage/scenes/event-guests/EventGuests.js]: If there are new `guests`

File: src/js/components/help/components/pages/backstage/scenes/event-settings/EventSettiings.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import styled from 'styled-components'
+import Sidebar from '../../../../help-topics'
+import Topic from '../../../../topic'
+import Desrciption from '../../../../help-description'
+import Description from '../../../../help-description';
+import {Link} from 'react-router-dom'
+
+const StyledEventGuests=styled.div`
```
Comment (innocentamadi) [inline src/js/components/help/components/pages/backstage/scenes/event-settings/EventSettiings.js]: Space around `=`

File: src/js/components/help/components/pages/backstage/scenes/event-settings/EventSettiings.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+`
+const EventPlanning = ()=>{
+  return(
+    <StyledEventGuests>
+      <div className="help-description-container">
+        <div ><Sidebar/></div> 
+        <div className="help-main-content"> 
+          <Topic>Event Settings</Topic>
+          <Description>
+            To edit you event, <Link to=""> go backstage</Link> then click on the settings.
```
Comment (innocentamadi) [inline src/js/components/help/components/pages/backstage/scenes/event-settings/EventSettiings.js]: To edit `your` event.

Also, where is this supposed to link to?

File: src/js/components/help/components/pages/communities/scenes/announcements/Announcements.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+const StyledAnnouncements = styled.div`
+.help-description-container{
+  display:flex;
+  margin 5px;
+  padding: 5px;
+}
+
+img{
+width:700px;
+height: 500px;
```
Comment (innocentamadi) [inline src/js/components/help/components/pages/communities/scenes/announcements/Announcements.js]: Indenting here should be fixed.

Comment (innocentamadi) [reply src/js/components/help/components/pages/communities/scenes/announcements/Announcements.js]: Also I noticed most of the help pages have similar styles. If that's the case, you should abstract the styling to a different component and just import that one component. We could discuss more on this later.
